src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -877,34 +877,36 @@ public void readFromNBT(final CompoundNBT data) {
         this.getFacadeContainer().readFromNBT(data);
     }
 
-    public List<ItemStack> getDrops(final List<ItemStack> drops) {
+    /**
+     * Append the contents of the parts to the drop list, e.g. crafting patterns but not the parts or facades
+     * themselves.
+     */
+    public void appendPartContentDrops(final List<ItemStack> drops) {
         for (final AEPartLocation s : AEPartLocation.values()) {
             final IPart part = this.getPart(s);
             if (part != null) {
-                drops.add(part.getItemStack(PartItemStack.BREAK));
                 part.getDrops(drops, false);
             }
-
-            if (s != AEPartLocation.INTERNAL) {
-                final IFacadePart fp = this.getFacadeContainer().getFacade(s);
-                if (fp != null) {
-                    drops.add(fp.getItemStack());
-                }
-            }
         }
-
-        return drops;
     }
 
-    public List<ItemStack> getNoDrops(final List<ItemStack> drops) {
+    /**
+     * Append the parts and facades to the drop list, but not their contents.
+     */
+    public void appendPartStacks(final List<ItemStack> drops) {
```

Comment:
Same here, the part itself should really handle it and not hide the drops somewhere else. But again API break, so more for 1.17